engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html

```diff
@@ -0,0 +1,156 @@
+{@boolean renderSection1=true}
+{@boolean renderSection2=true}
+{@boolean renderButtonSection2=true}
+{@boolean hideSubtitleSection2=true}
+{#include Common/insightsEmailBody}
+{#extra-css}
+    .rh-incident {
+      border-radius: 1em;
+      border: thin black solid;
+      background-color: #fac8c8;
+      padding: 0 0.5em;
+    }
+    .fa-exclamation-circle {
+        vertical-align: -2px;
+    }
+    .fa-exclamation-circle path {
+      d: path("M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z");
+      fill: rgb(201,25,11);
+    }
+{/extra-css}
+{#content-title}
+    Daily digest - Advisor - Red Hat Enterprise Linux
+{/content-title}
+{#content-title-section1}
+    Recommendation{#if action.context.advisor.total_recommendation > 1}s{/if}
+{/content-title-section1}
+{#content-subtitle-section1}
+    {action.context.advisor.total_incident} total incident{#if action.context.advisor.total_incident > 1}s{/if}
+{/content-subtitle-section1}
+{#content-title-right-part-section1}
+    <a target="_blank" href="{environment.url}/insights/advisor">{action.context.advisor.total_recommendation}</a>
+{/content-title-right-part-section1}
+{#content-body-section1}
+{#if action.context.advisor.new_recommendations.orEmpty.size > 0}
+    <table class="rh-data-table rh-m-bordered">
+        <thead>
+        <tr>
+            <th style="width: 60%; padding-left: 26px;">&nbsp;New Recommendation{#if action.context.advisor.new_recommendations.orEmpty.size > 1}s{/if}</th>
+            <th style="width: 20%">Total Risk</th>
+            <th style="width: 20%">Systems</th>
+        </tr>
+        </thead>
+        <tbody>
+        {#for rec in action.context.advisor.new_recommendations.keySet()}
+        <tr>
+            <td style="width: 60%; word-wrap: anywhere;">
+                {#let isIncident = action.context.advisor.new_recommendations.get(rec).has_incident.or("false")}
+                    <svg class="fa-exclamation-circle" height="16" width="16" viewBox="0 0 512 512" aria-hidden="true" role="img" {#if !isIncident || (true != isIncident && "true" != isIncident)} style='visibility: hidden'{/if}>
+                        <path><title>Incident</title></path>
+                    </svg>
+                {/let}
```

Comment:
Can we use an `if` from Qute to make things simpler than the current `let` + `visibility: hidden` logic?